downstream/modules/platform/proc-gw-define-rules-triggers.adoc

```diff
@@ -18,32 +18,22 @@ Always:: The trigger should always be fired.
-
-* * * *contains*: The specified character sequence in *Value* is contained within the attributes value returned from the source. For example, given an attribute value of ‘John’ from the source the contains *Comparison* would set the trigger to *True* if the trigger *Value* was set to ‘Jo’ and False if the trigger *Value* was ‘Joy’.
-
-* * * *matches*: The *Value* on the trigger is treated as a python regular expression and does an link::https://docs.python.org/3/library/re.html#re.match[re.match] (with case ignore on) between the specified *Value* and the value returned from the source system. For example, if the trigger's *Value* was ‘Jo’ the trigger would return *True* if the value from the source was John or Joanne or any other value which matched the regular expression ‘Jo’. The trigger would return False if the sources value for the attribute was ‘Dan’ because ‘Dan’ does not match the regular expression ‘Jo’. 
-
-* * * *ends with*: The trigger will see if the value provided by the source ends with the specified *Value* of the trigger. For example, if the source provided a value of ‘John’ the trigger would be True if its *Value* was set to ‘n’ or ‘on’. The trigger would be False if its *Value* was set to ‘z’ because the value ‘John’ coming from the source does not end with the value ’z’ specified by the trigger.
-
-* * * *equal*: The trigger will see if the value provided by the source is equal to (in its entirety) the specified *Value* of the trigger. For example, if the source returned the value ‘John’, the trigger would be True if its Value was set to ‘John’. Any value other than ‘John’ returned from the source would set this trigger to False.
-
-* * * *in*: The *in* condition will see if the value matches one of several values. When in is specified as the *Comparison* the *Value* field can be a comma separated list. For example, if a trigger had a *Value* of ‘John,Donna’ the trigger would be True if the attribute coming from the source had either the value ‘John’ or ‘Donna’ and False otherwize.
-
-* * The value that a users attribute will be matched against based on the *Comparison* field. See examples in the *Comparison* section above for examples. 
-
-NOTE
++
+* *Operation:* This field includes conditional settings that trigger the handling of the rule based on the specified *Groups* criteria. The choices include *and* and *or*. For example, if you select and the user logging in must be a member of all of the groups specified in the *Groups* field for this trigger to be true. Alternatively, if you select *or* the user logging in must be a member of any of the specified *Groups* in order for the trigger to fire. *NOTE*: If you are only keying off one group it doesn’t matter if you select and or or.
```

Comment:
at the end of this line, "and or or" looks like a mistake. Maybe format as "and" or "or".

Reply:
Fixed